packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -1529,11 +1531,11 @@ bool ArbCore::threadBody(ThreadDataStruct& thread_data) {
             auto add_status = addMessages(message_data, thread_data.cache);
             if (!add_status.status.ok()) {
                 thread_data.add_messages_failure_count++;
-                message_data_error_string = add_status.status.ToString();
-                message_data_status = MESSAGES_ERROR;
```

Comment:
Is the theory that this was what caused the error?

Reply:
It is definitely an error, not sure if it was the one we were hoping to fix